src/Raven.Client/Documents/Session/IDocumentQueryCustomization.cs

```diff
@@ -124,16 +124,5 @@ public interface IDocumentQueryCustomization
         /// </param>
         /// <param name="waitTimeout">Maximum time to wait for index query results to become non-stale before exception is thrown.</param>
         IDocumentQueryCustomization WaitForNonStaleResultsAsOf(long cutOffEtag, TimeSpan waitTimeout);
-
-        /// <summary>
-        ///     Instructs the query to wait for non stale results as of now.
-        /// </summary>
-        IDocumentQueryCustomization WaitForNonStaleResultsAsOfNow();
```

Comment:
Description/Summary from this should be moved to WaitForNonStaleResults?

Reply:
We can do it with little changes